app-misc/dfshow/metadata.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+<maintainer type="person">
+	<email>pepitsizous@zoho.com</email>
+	<description>Primary maintainer</description>
```

Comment:
This description is useless. Please add the proxy-maint project here below you, you need someone with @gentoo.org address to commit for you.